defaults/main.yml

```diff
@@ -11,6 +11,7 @@ postfix_mailname: "{{ ansible_fqdn }}"
 postfix_aliases: []
 postfix_virtual_aliases: []
 postfix_sender_canonical_maps: []
+postfix_recipient_canonical_maps: []
 postfix_generic: []
```

Comment:
Please add `postfix_header_checks` here too

Reply:
`postfix_header_checks: []`

Reply:
done in newest commit